01_02_Combat_Rules.txt

```diff
@@ -888,12 +888,31 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
 health. Any additional attacks you sustain will push you into negative health. 
 
 Stabilize:
-	As an action, a teammate may bring you back up to 1 health point by succeeding 
-on a DC 80 Medicine skill check to stabilize you. Medicine skill, INT mod, and any
-bonuses from medical equipment such as medic-kits are added to the roll. Once stabilized,
-you may take actions  and speak as normal.
-
-	Don't forget that a recently stabilized character may need  to take actions to
+	As an action, a character may choose an adjacent, Downed character and 
```

Comment:
Nonblocking; I think we should lengthen the time it takes to stabilize a character so that it's longer than the time to raise and fire a weapon. Perhaps I should also lengthen the time for the stabilize Doctor Procedure.

Reply:
I concur, but if we change one we should change the other too, and that makes it scope creep I think?
If we decide on longer than 2 actions, we can also delete the bit I put in here about reusing repair/medkits.